canonical/ght
# Replicate and delete a specific job post (WD-7364)

## Done
- Replicate and delete job post - interactively or non-interactively - with the `repost` command.

## QA
- Check that a specific job post is cloned and deleted by running `yarn dev repost -i` (for interactive).
- Check that a specific job post is cloned and deleted by running `yarn dev repost <JOB_POST_ID>` (for non-interactive).

## Issue
https://github.com/canonical/ght/issues/204
